src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import javax.sound.midi.SysexMessage;
+import java.util.Scanner;
+
 public class Duke {
+
+    static Task[] tasks = new Task[100];
+    static int taskPosition = 0;
+
+
+    public static void storeTask(Task t){
```

Comment:
Easy to understand what this function does by the function name. Good use of the naming. I think to improve on the naming and readability further, the parameter t can be renamed to maybe newTask. This can be easily understood that the function storeTask is storing the newly created task into a certain position on the tasks array. Lastly, you will be using newTask.description instead of t.description.